src/simulation.rs

```diff
@@ -141,6 +142,45 @@ impl Simulation {
         // wait for report response
         let final_report = events_report.await.expect("events collection to end");
 
+        let ready_users = self.entities.values().filter_map(|entity| {
+            if let Entity::Ready { user } = entity {
+                Some(user)
+            } else {
+                None
+            }
+        });
+
+        let channel_stats = ready_users.clone().fold(
+            (0, usize::MAX, 0, HashSet::new()),
+            |(max, min, total_chans_of_users, mut channels_created), user| {
+                let u = user.try_read();
+
+                if let Ok(u) = u {
+                    log::debug!("getting user {} channel stats", u.localpart);
+                    let results = u.get_user_channels_stats((
+                        max,
+                        min,
+                        total_chans_of_users,
```

Comment:
What's the idea of this number, do we want the number of joins or the number of unique channels joined?

Reply:
`channels_created` refers to unique channels joined (i should change the name) and `total_chans_of_user` is the sum of each user channel (joined) count to be able to calculate an average.